accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,360 @@
+    - `GetExtensionFunctionPointer`
+    - `GetQos`
+    - `GetGroupQos`
+    - `MultipointLoopback`
+    - `MulticastScope`
+    - `SetQos`
+    - `SetGroupQos`
+    - `TranslateHandle`
+    - `RoutingInterfaceQuery`
+    - `RoutingInterfaceChange`
+    - `AddressListQuery`
+    - `AddressListChange`
+    - `QueryTargetPnpHandle`
+    - `NamespaceChange`
+    - `AddressListSort`
+    - `ReceiveAll`
+    - `ReceiveAllMulticast`
+    - `ReceiveAllIgmpMulticast`
+    - `KeepAliveValues`
+    - `AbsorbRouterAlert`
+    - `UnicastInterface`
+    - `LimitBroadcasts`
+    - `BindToInterface`
+    - `MulticastInterface`
+    - `AddMulticastGroupOnInterface`
+    - `DeleteMulticastGroupFromInterface`
+* **Socket**
+    - `SetIPProtectionLevel(IPProtectionLevel)`
+* **TcpListener**
+    - `AllowNatTraversal(Boolean)`
+* **TransmitFileOptions**
+    - `UseKernelApc`
+    - `UseSystemThread`
+    - `WriteBehind`
+* **UdpClient**
+    - `AllowNatTraversal(Boolean)`
+
+### System.Runtime.InteropServices
+
+* **DispatchWrapper**
+    - `get_WrappedObject()`
+* **Marshal**
+    - `AddRef(IntPtr)`
+    - `BindToMoniker(String)`
+    - `ChangeWrapperHandleStrength(Object, Boolean)`
+    - `CreateAggregatedObject(IntPtr, Object)`
+    - `CreateAggregatedObject<T>(IntPtr, T)`
+    - `FinalReleaseComObject(Object)`
+    - `GetComInterfaceForObject(Object, Type)`
+    - `GetComInterfaceForObject(Object, Type, CustomQueryInterfaceMode)`
+    - `GetComInterfaceForObject<T, TInterface>(T)`
+    - `GetIUnknownForObject(Object)`
+    - `GetNativeVariantForObject(Object, IntPtr)`
+    - `GetNativeVariantForObject<T>(T, IntPtr)`
+    - `GetObjectForIUnknown(IntPtr)`
+    - `GetObjectForNativeVariant(IntPtr)`
+    - `GetObjectForNativeVariant<T>(IntPtr)`
+    - `GetObjectsForNativeVariants(IntPtr, Int32)`
+    - `GetObjectsForNativeVariants<T>(IntPtr, Int32)`
+    - `GetStartComSlot(Type)`
+    - `GetTypedObjectForIUnknown(IntPtr, Type)`
+    - `GetTypeFromCLSID(Guid)`
+    - `GetTypeInfoName(ITypeInfo)`
+    - `GetUniqueObjectForIUnknown(IntPtr)`
+    - `QueryInterface(IntPtr, Guid, IntPtr)`
+    - `Release(IntPtr)`
+    - `ReleaseComObject(Object)`
+
+### System.Security.Cryptography
+
+* **CspKeyContainerInfo**
+    - `get_Accessible()`
+    - `get_Exportable()`
+    - `get_HardwareDevice()`
+    - `get_KeyContainerName()`
+    - `get_KeyNumber()`
+    - `get_MachineKeyStore()`
+    - `get_Protected()`
+    - `get_ProviderName()`
+    - `get_ProviderType()`
+    - `get_RandomlyGenerated()`
+    - `get_Removable()`
+    - `get_UniqueKeyContainerName()`
+* **DSACryptoServiceProvider**
+    - `get_CspKeyContainerInfo()`
+    - `ImportCspBlob(Byte[])`
+* **PasswordDeriveBytes**
+    - `CryptDeriveKey(String, String, Int32, Byte[])`
+* **RC2CryptoServiceProvider**
+    - `set_UseSalt(Boolean)`
+* **RSACryptoServiceProvider**
+    - `.ctor(CspParameters)`
+    - `.ctor(Int32, CspParameters)`
+    - `get_CspKeyContainerInfo()`
+    - `ImportCspBlob(Byte[])`
```

Comment:
looks like it is supported: https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/RSACryptoServiceProvider.Unix.cs#L135-L139

```suggestion
```